src/core/net/dhcp6_client.cpp

```diff
@@ -76,10 +76,8 @@ void Dhcp6Client::UpdateAddresses(void)
     NetworkData::OnMeshPrefixConfig config;
 
     // remove addresses directly if prefix not valid in network data
-    for (size_t i = 0; i < OT_ARRAY_LENGTH(mIdentityAssociations); i++)
+    for (IdentityAssociation &ia : mIdentityAssociations)
```

Comment:
```suggestion
    for (IdentityAssociation &idAssocication : mIdentityAssociations)
```

Reply:
I used `ia` because there are other places that are also using `ia`. 
Would it be fine since DHCPv6 developers should just know what `ia` stands for well?